src/cypress/integration/Form/Checkbox/checkbox-disabled.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { checkboxCopy } from 'shared/data/copyContent';
+
+describe('checkbox-disabled-test', () => {
+  before(() => {
+    cy.visitStorybook();
+    cy.loadStory('components-form-checkbox', 'disabled');
+  });
+
+  it('Should be displayed', () => {
+    cy.findByText(checkboxCopy.disabled);
+  });
+
+  it('Should be disabled', () => {
+    cy.findByText(checkboxCopy.disabled)
+      .siblings('input')
+      .should('be.disabled');
+  });
+
+  it('Should be not be checked', () => {
+    cy.findByText(checkboxCopy.disabled)
+      .siblings('input')
+      .should('not.be.checked');
+  });
+
+  it('Should be checked', () => {
```

Comment:
Let's update these to be more specific. Something like:
* Should have both options disabled
* Should not have the first option checked
* Should have the second option checked